localMatchApp/sources/events.cpp

```diff
@@ -0,0 +1,81 @@
+#include "../headers/events.h"
```

Comment:
Try just using "events.h", if it does not work, at the directory in the cmake file